### PR TITLE
duplicate bookmark negative test fix

### PR DESCRIPTION
**Pull request by pondrejk**
### Problem Statement
With https://github.com/SatelliteQE/airgun/pull/1200 in, another problem uncovered in bookmarks. Trying to create a bookmark with already existing name doesn't raise exception in old style dialog (wontfix bz 1992652), it does so in the new style dialog (different one in stream than in 6.x runs). Affects ~20 tests 

### Solution
this pr attempts to cover all the cases

### CODE REVIEW

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_bookmarks.py -k test_negative_create_with_duplicate_name

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 5911
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_bookmarks.py -k test_negative_create_with_duplicate_name --external-logging
Test Result : = 10 failed, 28 passed, 156 deselected, 138 warnings, 1 error in 2309.84s (0:38:29) =
```

File: tests/foreman/ui/test_bookmarks.py [hunk 1]
```diff
@@ -259,10 +259,15 @@ def test_negative_create_with_duplicate_name(session, ui_entity, module_target_s
...
-        # assert there are no duplicate bookmarks
-        new_search = session.bookmark.search(bookmark.name)
-        assert len(new_search) == 1
-        # assert bookmark query wasn't overriden
-        assert new_search[0]['Search query'] == existing_bookmark['Search query']
+        # this fails but does not raise UI error in old style dialog, BZ#1992652 closed
+        # wontfix, but new style dialog raises error, both situations occur
+        try:
+            ui_lib.create_bookmark({'name': bookmark.name, 'query': query, 'public': True})
+        except (DisabledWidgetError, NoSuchElementException):
```
Comment (lhellebr) [inline tests/foreman/ui/test_bookmarks.py:266]: So which dialogue will be used? Shouldn't we expect one of them? Shouldn't we expect an exception depending on what dialogue is being used? This looks too vague.

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_bookmarks.py -k test_negative_create_with_duplicate_name

File: tests/foreman/ui/test_bookmarks.py (hunk 1 above)
Comment (pondrejk) [reply tests/foreman/ui/test_bookmarks.py:266]: > So which dialogue will be used? 

@lhellebr Hm, I didn't find a way to easily establish that, from the airgun point of view, they are the same. The differentiator is the behavior when trying to save bookmark with existing name

> Shouldn't we expect an exception depending on what dialogue is being used? This looks too vague.

In a way, I'm doing just that. If it doesn't raise, then I assume old style dialog and do additional checks to make sure duplicate stuff wasn't created.

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_bookmarks.py -k test_negative_create_with_duplicate_name

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_bookmarks.py -k test_negative_create_with_duplicate_name

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6041
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_bookmarks.py -k test_negative_create_with_duplicate_name --external-logging
Test Result : = 28 failed, 8 passed, 1 skipped, 156 deselected, 118 warnings, 2 errors in 1921.89s (0:32:01) =
```

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_bookmarks.py -k test_negative_create_with_duplicate_name

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6052
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_bookmarks.py -k test_negative_create_with_duplicate_name --external-logging
Test Result : = 8 failed, 28 passed, 1 skipped, 156 deselected, 124 warnings, 2 errors in 2520.19s (0:42:00) =
```

**Review by lhellebr — APPROVED** [review]
We discussed this personally and the way it is now looks good

**Pull request merged**